doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1577,62 +1615,188 @@ Response code:
          }
         ]
 
-### updateContext [POST /v2/updateContext]
+### Register [POST /v2/op/register]
 
+This operation allows to create, update and/or delete several registrations in a single batch operation.
 The payload is an object with two properties:
 
-* `action`, to specify the kind of updateContext action to do (see OMA NGSI specification for the semantics),
-   either UPDATE, APPEND or DELETE
-* `entities`, an array of entities, each one specified using the the JSON entity representation format
-  (described in a section above).
+* `actionType`, to specify the kind of register action to do: either APPEND, UPDATE, DELETE.
+* `registrations`, an array of registration, each one specified using the JSON registration representation
+  format (described in a section above). In the case of APPEND operation, the registration `id` must not be
+  included.
 
 Response:
 
-* Successful operation uses 201 Created.
+* Successful UPDATE or DELETE operation uses 204 No Content.
+* Successful APPEND operation uses 201 Created and returns a list of IDs, each one corresponding to the ID of the
+  element in the request payload in the same order.
 * Errors use a non-200 and (optionally) an error payload. See subsection on Error Responses for more details.
 
 + Request (application/json)
 
         {
-            "action": "APPEND",
-            "entities": [
-                {
-                    "type": "Room",
-                    "id": "Bcn-Welt",
-                    "temperature": 21.7,
-                    "humidity": 60
-                },
-                {
-                    "type": "Room",
-                    "id": "Mad_Aud",
-                    "temperature": 22.9,
-                    "humidity": 85
-                }
-            ]
+          "actionType": "APPEND",
+          "registrations: [
+             {
+               "subject": {
+                  "type": "Room",
+                  "attributes": [
+                   "humidity"
+                 ]
+               },
+               "callback": "http://localhost:1234",
+               "duration": "PT1M"
+             },
+             {
+               "subject": {
+                 "type": "Car",
+                 "attributes": [
+                   "speed"
+                 ]
+               },
+               "callback": "http://localhost:5678",
+               "duration": "PT1M"
+             }
+          ]
         }
 
 + Response 201
 
-    + Headers
+        [
+          "abcd",
+          "efgh"
+        ]
 
-            Location: /v2/entities/Bcn-Welt
+### Discover [POST /v2/op/discover]
+
```

Comment:
Issues created: https://github.com/telefonicaid/fiware-orion/issues/1145 and https://github.com/telefonicaid/fiware-orion/issues/1146